dotcom-rendering/src/components/ElementContainer.tsx

```diff
@@ -34,6 +34,21 @@ const setBackgroundColour = (colour: string) => css`
 	background-color: ${colour};
 `;
 
+const timelineStyles = (colour: string) => css`
+	position: relative;
+	margin: auto;
+
+	${from.leftCol} {
+		max-width: 1140px;
+		border-left: 1px solid ${colour};
+		border-right: 1px solid ${colour};
```

Comment:
Would the use of `--timeline-event-border` not be appropriate here? It brings about several benefits:
- no dynamic generation of Emotion styles
- consistency across all formats

It might require quite a bit more work we can address as a follow-up, as I appreciate the logic for overrides is a bit finnicky and this may not be the best PR to do this :D